smart_card_connector_app/src/chrome-api-provider.js

```diff
@@ -0,0 +1,62 @@
+goog.provide('GoogleSmartCard.ChromeAPIProvider');
+
+goog.require('GoogleSmartCard.PcscLiteServerClientsManagement.ServerRequestHandler');
+
+goog.scope(function() {
+
+const GSC = GoogleSmartCard;
+
+/**
+ * @constructor
+ * @extends goog.Disposable
+ */
+GSC.ChromeAPIProvider = function(serverMessageChannel, serverReadinessTracker) {
```

Comment:
nit: ES6 class is preffered in new code. See, e.g., commit 36ffa0d104a779bc5108cfe468814fd6d6394e08.